distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -87,10 +87,8 @@ type Vector a
 
              Vector.fill length=50 item=42
     fill : Integer -> Any -> Vector Any
-    fill length ~item =
```

Comment:
Turns out removing `~` caused this regression https://github.com/enso-org/enso/pull/10142#discussion_r1623219874, CCing @radeusgd and @jdunkerley 

Previously the `~item` was evaluated again and again for each element of the array. Now it is evaluated just once.

Reply:
Personally I find the new `fill` semantic better. Use `Vector.new` if you want to create vector with different values.

Reply:
> Personally I find the new `fill` semantic better. Use `Vector.new` if you want to create vector with different values.

Yes, I too think that current semantics is better.